selfdrive/boardd/boardd.cc

```diff
@@ -260,8 +261,8 @@ void can_recv_thread() {
     uint64_t cur_time = nanos_since_boot();
     int64_t remaining = next_frame_time - cur_time;
     if (remaining > 0){
-      useconds_t sleep = remaining / 1000;
-      usleep(sleep);
+      int sleep = remaining / 1000;
+      std::this_thread::sleep_for(std::chrono::microseconds(sleep));
```

Comment:
If we're using std::chrono here we can also get rid of `sleep` and do `      std::this_thread::sleep_for(std::chrono::nanoseconds(remaining));`